pubsub/kafka/kafka.go

```diff
@@ -62,6 +64,46 @@ func (p *PubSub) Subscribe(ctx context.Context, req pubsub.SubscribeRequest, han
 	return p.kafka.Subscribe(p.subscribeCtx)
 }
 
+func (p *PubSub) BulkSubscribe(ctx context.Context, req pubsub.SubscribeRequest, handler pubsub.BulkHandler) error {
+	p.kafka.AddTopicBulkHandler(req.Topic, adaptBulkHandler(handler))
+	maxBulkCount, err := strconv.Atoi(req.Metadata["maxBulkCount"])
+	if err != nil {
+		maxBulkCount = 20
```

Comment:
Can we set these magic numbers somewhere else?

Reply:
metadata.go seems to have metadata specific to the one given in kafka component yaml. These metadata are provided per topic based in response to "/dapr/subscribe", created a struct in internal/component/kafka/kafka.go, as it already has handlers per topic based.